### PR TITLE
INTYGFV-15843: Implemented api to fetch persons by personId. Uses "fakepu" which fetches persons from provided json-resources instead of calling PU.

### PR DESCRIPTION
**Pull request by mww-sopra-steria**

### CODE REVIEW

File: intyg-proxy-service/integration-fakepu/src/test/java/se/inera/intyg/intygproxyservice/integration/fakepu/FakePuIntegrationServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    @BeforeEach
+    void setUp() {
+      doReturn(null)
+          .when(fakePuRepository)
+          .getPerson(PERSON_ID);
+    }
+
+    @Test
+    void shallReturnStatusFoundWhenPersonExists() {
```
Comment (ChrilleSigerhed) [inline intyg-proxy-service/integration-fakepu/src/test/java/se/inera/intyg/intygproxyservice/integration/fakepu/FakePuIntegrationServiceTest.java:77]: ```shallReterunStatusNotFoundWhenPersonDontExist```?

File: intyg-proxy-service/integration-pu/src/main/java/se/inera/intyg/intygproxyservice/integration/pu/PuIntegrationService.java [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
+import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Service;
 import se.inera.intyg.intygproxyservice.integration.api.pu.Person;
 import se.inera.intyg.intygproxyservice.integration.api.pu.PuRequest;
 import se.inera.intyg.intygproxyservice.integration.api.pu.PuResponse;
 import se.inera.intyg.intygproxyservice.integration.api.pu.PuService;
 
 @Service
+@Profile("!fakepu")
```
Comment (ChrilleSigerhed) [inline intyg-proxy-service/integration-pu/src/main/java/se/inera/intyg/intygproxyservice/integration/pu/PuIntegrationService.java:11]: Consider creating a constant for fakepu to make it easier to track its usage.

**Review by ChrilleSigerhed — APPROVED** [review]
Awsome, looks great!

File: intyg-proxy-service/integration-pu/src/main/java/se/inera/intyg/intygproxyservice/integration/pu/PuIntegrationService.java (hunk 2 above)
Comment (mww-sopra-steria) [reply intyg-proxy-service/integration-pu/src/main/java/se/inera/intyg/intygproxyservice/integration/pu/PuIntegrationService.java:11]: Thanks for the good suggestion! I will introduce this in the next PR!

**Pull request merged**